symtabAPI/src/debug.C

```diff
@@ -35,7 +35,7 @@
 #include "debug.h"
 
 int sym_debug_create = 0;
-int sym_debug_parsing = 0;
+int sym_debug_parsing = 1;
```

Comment:
Let's not enable this by default. It's quite noisy.

Reply:
Restore to default = 0.